bin/ch-test

```diff
@@ -55,7 +55,7 @@ Usage:
   $ $(basename "$0") [PHASE] [--scope SCOPE] [--pack-fmt FMT] [OPTIONS]
 
 Valid phases: build, build-images, run, examples, all, mk-perm-dirs, clean,
-              rm-perm-dirs
+              rm-perm-dirs, rootemu, most
```

Comment:
I'd list these in order of when they happen. Maybe it's time to explain a bit more carefully because some of those phases contain other phases?